web/client-api/src/main/java/io/deephaven/web/client/api/i18n/JsTimeZone.java

```diff
@@ -14,120 +14,538 @@
     // Cache the time zones that are parsed so we don't need to recalculate them
     private static final Map<String, JsTimeZone> timeZoneCache = new HashMap<>();
 
+    // Map of all time IDs to their json code string
+    private static final Map<String, TimeZone> timeZones = new HashMap<>();
```

Comment:
Why not use the existing timeZoneCache map (but rename it probably), and prepopulate all the JsTimeZone objects? We're already doing the "hard" thing up front without the benefit of caching, why have the cache hold just a tiny object with a reference to the big, already created object?

I suggest dropping the new field timeZones, and just rename timeZoneCache to timeZones. Then, the timeZones.put() methods would do the tiny bit of extra wrapping up front to make a JsTimeZone instance